content/administration/maintain/domain_names.rst

```diff
@@ -0,0 +1,309 @@
+   - `Namecheap <https://www.namecheap.com/support/knowledgebase/article.aspx/9646/2237/how-to-create-a-cname-record-for-your-domain>`_
+   - `OVH <https://docs.ovh.com/us/en/domains/web_hosting_how_to_edit_my_dns_zone/#add-a-new-dns-record>`_
+   - `CloudFlare <https://support.cloudflare.com/hc/en-us/articles/360019093151>`_
+   - `Google Domains <https://support.google.com/domains/answer/3290350?hl=en>`_
+
+.. _domain-name/naked-domain:
+
+Naked domain
+~~~~~~~~~~~~
+
+A **naked domain** is a domain name that doesn't have any subdomain at the beginning of the address
+(e.g., ``odoo.com`` instead of ``www.odoo.com``).
+
+You may want your naked domain to point to your website as some visitors may not type the full
+domain name to access your website.
+
+#. Open your domain name's manager dashboard.
+#. Create a **redirection** from the naked domain (``example.com``) to your main domain name
+   (``www.example.com``)
+
+.. note::
+   Depending on your domain name registrar, this redirection may be already pre-configured.
+
+.. _domain-name/db-map:
+
+Map your domain name with your Odoo database
+--------------------------------------------
+
+.. tabs::
+
+   .. group-tab:: Odoo Online
+
+      Open your `database manager <https://www.odoo.com/my/databases>`_, click on the *settings*
+      button next to your database, on *Domain names*, and then on *Use my own domain* at the bottom
+      of the right column.
+
+      Type the domain name you want to add to this database, then click on *Verify* to check if the
+      the CNAME record is correctly configured. Once done, click on *I confirm, it's done*.
+
+      .. image:: domain_names/online-map.png
+         :align: center
+         :alt: Verification of the CNAME records of a domain name before mapping it with a database
+
+   .. group-tab:: Odoo.sh
+
+      Go to :menuselection:`Branches --> select your branch --> Settings --> Custom domains`, type
+      the domain name you want to add to this database, then click on *Add domain*.
+
+      .. image:: domain_names/odoo-sh-map.png
+         :align: center
+         :alt: Mapping a domain name with an Odoo.sh branch
+
+      .. seealso::
+         - :ref:`Odoo.sh branches: settings tab <odoosh-gettingstarted-branches-tabs-settings>`
+
+.. warning::
+   Make sure to :ref:`add a CNAME record <domain-name/cname>` to your domain name's DNS **before**
+   mapping your domain name with your Odoo database.
+
+   Failing to do so may impede the validation of the :ref:`SSL certificate <domain-name/ssl>` and
+   would result in a *certificate name mismatch* error. This is often displayed by web browsers as a
+   warning such as *"Your connection is not private"*.
+
+   If this is your case and have added your domain name to your database's settings less than five
+   days ago, wait 24 hours as the validation may still happen. Otherwise, please `submit a support
+   ticket <https://www.odoo.com/help>`_ including screenshots of your CNAME records.
```

Comment:
```suggestion
   If this is the case and you have added the domain name to your database's settings less than five
   days ago, wait 24 hours as the validation may still happen. Otherwise, please `submit a support
   ticket <https://www.odoo.com/help>`_ including screenshots of your CNAME records.
```